code/modules/mob/living/carbon/emote.dm

```diff
@@ -243,6 +244,19 @@
 	male_sounds =  list('sound/misc/malescream1.ogg', 'sound/misc/malescream2.ogg', 'sound/misc/malescream3.ogg', 'sound/misc/malescream4.ogg', 'sound/misc/malescream5.ogg', 'sound/misc/wilhelm.ogg', 'sound/misc/goofy.ogg')
 	female_sounds = list('sound/misc/femalescream1.ogg', 'sound/misc/femalescream2.ogg', 'sound/misc/femalescream3.ogg', 'sound/misc/femalescream4.ogg', 'sound/misc/femalescream5.ogg')
 	sound_message = "screams in agony!"
+	mob_type_blacklist_typelist = list(/mob/living/carbon/human/vox, /mob/living/carbon/human/skelevox)
```

Comment:
This won't work, because someone can be a vox or skelevox without being these types, due to how `species` works.
Actually most vox and skelevox aren't these types.

Reply:
I've tested locally. It works interestingly enough. Given, I could probably use isspecies too. But it'd be a bit of a pain to add in specific snowflake vars for a specific set of emotes when there's an already available set of vars available.

Reply:
Ah. I see. Fixing it.

Reply:
Fixed. I think.